torch/_inductor/ir.py

```diff
@@ -4185,21 +4204,30 @@ def __repr__(self):
 
         tensor_args = [Shim(x.codegen_reference()) for x in self.inputs]
         args, kwargs = self.unflatten_args(tensor_args, self.constant_args)
-        args = [V.graph.wrapper_code.val_to_arg_str(x) for x in args]
-        if (
-            V.graph.cpp_wrapper
-            and hasattr(self, "args_default_value")
-            and not config.is_fbcode()
-        ):
-            n_args = len(args)
-            n_pos_args = len(self.args_default_value)
-            # Some positional args are not provided, need to use their default value in cpp wrapper
-            if n_args < n_pos_args:
-                pos_args = [
-                    self.get_arg_default_value(i) for i in range(n_args, n_pos_args)
-                ]
-                pos_args = [V.graph.wrapper_code.val_to_arg_str(x) for x in pos_args]
-                args.extend(pos_args)
+        if not V.graph.cpp_wrapper:
+            args = [V.graph.wrapper_code.val_to_arg_str(x) for x in args]
+        else:
+            args = [
+                V.graph.wrapper_code.val_to_cpp_arg_str(
+                    param.real_type, x, self.is_legacy_abi_kernel()
+                )
+                for param, x in zip(self.op_overload._schema.arguments, args)
+            ]
+            if hasattr(self, "args_default_value") and not config.is_fbcode():
```

Comment:
Seems this PR is not rebased to the head of the master? This part is different from:

https://github.com/pytorch/pytorch/blob/main/torch/_inductor/ir.py#L4265-L4285

where ```config.is_fbcode()``` has been removed.